### PR TITLE
added the improvements/future plans in main

### PR DESCRIPTION
**Pull request by julie2045**
I can't seem to figure out why the battle won't display. I have tried but I don't why it won't work.

### CODE REVIEW

**Comment by l2lam** [comment]
Did you try deleting all the *.class files and re-running?

**Comment by julie2045** [comment]
> Did you try deleting all the *.class files and re-running?

Yes I tried that and it still won't display it.

File: Game.java [hunk 1]
```diff
@@ -97,11 +97,7 @@ void startGame() {
...
 
-    //battle begins, load each creature and health 
-      int round = 1;
-      System.out.println(String.format("Round %d:", round));
-      System.out.println(String.format("%s: %d hp | %s: %d hp", this.player1.species(), this.player1.health(), this.player2.species(), this.player2.health()));
-  
+
     // Start the battle rounds, continue until one is defeated
     int round = 1;
     while (!this.isGameOver()); {
```
Comment (l2lam) [inline Game.java:103]: OMG, you have a semi colon right after the while loop condition... the loop will not do anything.
```suggestion
    while (!this.isGameOver()) {
```

Comment (julie2045) [reply Game.java:103]: Okay now it works. Thanks!

**Pull request merged**